src/AdePTGeant4Integration.cpp

```diff
@@ -84,8 +84,46 @@ void Deleter::operator()(ScoringObjects *ptr)
 
 } // namespace AdePTGeant4Integration_detail
 
+std::vector<G4VPhysicalVolume const *> AdePTGeant4Integration::fglobal_vecgeom_to_g4_map;
+
 AdePTGeant4Integration::~AdePTGeant4Integration() {}
 
+std::vector<G4VPhysicalVolume const *> AdePTGeant4Integration::GetPhysicalVolumeMap()
+{
+  std::vector<const G4VPhysicalVolume *> vecgeomToG4Map;
```

Comment:
I would pass this by reference as an argument to avoid a full vector copy. Clearing and resizing it, of course.